app/src/ui/dispatcher/dispatcher.ts

```diff
@@ -2672,11 +2673,13 @@ export class Dispatcher {
       log.warn(
         '[cherryPick] Invalid Cherry-picking State: Could not determine selected commits.'
       )
+      this.endCherryPickFlow(repository)
```

Comment:
You will see these added in a few places. These are spots where we 'should' not every get into.. but if we did, we don't want to remain in a cherry-picking state cause it will lead to other weirdness (see drag overlay, or context menu cherry-pick disabled). Thus, put the endCherryPickFlow to more cleanly end on errors.